### PR TITLE
Second option for Issue #861

### PR DESCRIPTION
**Pull request by jgallimore**
This restyles the page a little, and allows multiple versions for each vendor.

### CODE REVIEW

**Comment by jgallimore** [comment]
This is second option for #861, presented on the marketing call today.

**Comment by autumnfound** [comment]
This is an awesome start, and it looks really clean! The only thing I might change in terms of styling is there is padding on the compatible images using the padding-left and padding-right classes. This means at mobile when there isn't much space we get really tiny images. 

If we add this back but as a media query CSS rule, we could get a bit more control and give the images a bit of room to breathe.
The following sample should add the padding back when your screen is wider than 768 px. 
```
@media (min-width: @screen-md-min) {
  .product-list .center-block {
    padding-left: 30px;
    padding-right: 30px;
  }
}
```

**Comment by jclingan** [comment]
Please remove references to "Full Profile" and "Full Release". The platform can be referred to as "Jakarta EE 8" (IMHO, preferred) or "Jakarta EE 8 Platform". "Jakarta EE 8 Web Profile" is the only "profile" in place today.

**Review by edbratt — CHANGES_REQUESTED** [review]
Visually, the numbered links below the logo/product name look, to me like download links, but they are just the TCK results pages. It would be nice if there was a way to clarify that element below the logo and product name is the TCK result -- and to get to the product download, you clink the image. All my opinion.
Please make the following changes:
- [ ] Change headline for Platform section -- Remove "Full", to read: "Jakarta EE 8 Platform Compatible Products"
- [ ] Change Eclipse GlassFish TCK Results link label to "5.1 Platform"

**Comment by jgallimore** [comment]
Thanks @jclingan @edbratt. I've made those changes and they should be visible on the Netlify preview once the build is complete.

**Comment by jgallimore** [comment]
Hi all,

I believe I'm up to date with all the requested changes. Is there anything blocking this from being merged? We've had this open for 18 days, so it would be good to get this over the finishing line.

**Comment by shabnammayel** [comment]
Can you please remove "Platform", "GA", "Final","Platform" and "Web Profile" that are displayed next to the versions? 
Screen Shot 2020-09-14 at 8 24 46 AM

Screen Shot 2020-09-14 at 8 25 41 AM

**Comment by jgallimore** [comment]
@shabnammayel Sure thing - I just pushed that change. That should be on the preview once the CI job completes. One thing I would say is that this is probably not an issue for Eclipse Glassfish, but might be an issue for Redhat JBoss EAP and WildFly. I believe the .GA is part of the version for the artifacts. They also have artifacts with the same number, but '.RC', for release candidates. This is something JBoss has done since at least 2005, if my memory serves me correctly: https://sourceforge.net/projects/jboss/files/JBoss/

@jclingan - any thoughts?

**Comment by TanjaObradovic** [comment]
+1 this looks good to me, but let's wait to hear from @jclingan

**Comment by kwsutter** [comment]
These pages look much better!  But, where will this be documented for the teams submitting the certifications?  We should be consistent with the version links going to specific TCK results, and the logo links can be to any vendor-specified location (main page, download page, specific version, etc).

And, what did we decide on the number of versions listed?  1, 2, 3, ???

**Comment by shabnammayel** [comment]
@kwsutter We are woking on updating the GitHub issue template on the "get listed"page with the new requirements. 
Yes up to 3 versions was approved with the option of adding "see all" and link to a page that includes all downloads in case we have more than 3. 
I think the GitHub template request should specify that for consistency both logo links for Full Platform and Web Profile should go to the same page selected by the vendor (main page, download page, specific version, etc).

**Comment by kwsutter** [comment]
When will we have this in place?  We still have the [second Open Liberty certification ](https://github.com/jakartaee/jakarta.ee/issues/855) that we would like to get recognized.  And, this would be a good "test" of this layout.

File: data/compatible_products.yml
Comment (kwsutter) [inline data/compatible_products.yml:96]: This should be:  https://openliberty.io/certifications/jakartaee/webprofile/8/TCKResults.html

File: data/compatible_products.yml
Comment (kwsutter) [inline data/compatible_products.yml:112]: This doesn't look right.  The Web Profile certification is pointing at the Full Platform TCK results?  Check with RH before merging.

File: data/compatible_products.yml
Comment (kwsutter) [inline data/compatible_products.yml:46]: This link doesn't look right.  The Full Platform certification is pointing at Web Profile results?  I'd check with RH before merging.

**Review by kwsutter — CHANGES_REQUESTED** [review]
A few links are not correct.  Let's get these corrected before we merge this update.  We want this revamping to be look correct out of the chute... (smile).  Thanks!

**Comment by jclingan** [comment]
.GA and .Final are formally part of the Product (JBoss EAP) and Opensource (WildFly) release tags. So, it depends on how formal you want to be. I'm ok either way, so just version numbers are fine.

**Comment by dblevins** [comment]
> .GA and .Final are formally part of the Product (JBoss EAP) and Opensource (WildFly) release tags. So, it depends on how formal you want to be. I'm ok either way, so just version numbers are fine.

Right.  We should use the release name as the vendor wants it to appear.  Sometimes the releases being certified are milestones, betas, previews or some other status that is significant to what the vendor is attempting to communicate to its users.

File: data/compatible_products.yml
Comment (jclingan) [reply data/compatible_products.yml:46]: Egad, good catch. That should be https://github.com/wildfly/certifications/blob/EE8/WildFly_18.0.0.Final/jakarta-full-platform.adoc

File: data/compatible_products.yml
Comment (jclingan) [reply data/compatible_products.yml:112]: The EAP web profile link is https://github.com/RedHatOfficial/jbosseap-certifications/blob/EE8/EAP-7.3.0/jakarta-web-profile.adoc

**Review by kwsutter — APPROVED** [review]
LGTM; Thanks!

**Comment by shabnammayel** [comment]
Is this ready to go live @jgallimore? I think it was approved on the MKT Committee call last week, right? I was not on the call last week. Was there any additional comments/suggestions?

**Comment by shabnammayel** [comment]
I think Apusic's product version was corrected to V10 https://github.com/jakartaee/jakarta.ee/pull/872 and the link to their product version (TCK results page) should be changes to http://www.apusic.com/info_safety/server/aas_v10_tck

**Comment by shabnammayel** [comment]
Thanks @jgallimore, appreciate it. @chrisguindon can we make this page live when you get a chance? thanks

**Pull request merged**
